index.js

```diff
@@ -2,48 +2,38 @@
 'use strict';
 
 const isFullwidthCodePoint = codePoint => {
-	if (Number.isNaN(codePoint)) {
```

Comment:
I would prefer to have:

```js
if (!Number.isInteger(codePoint)) {
	return false;
}
```

here.

Reply:
I did what you asked me to do, but for my personal knowledge, what is the advantage of separating the test: `!Number.isInteger(codePoint)`

Reply:
Just my preference, and it's slightly more readable. I prefer having type "assertions" separated from the actual logic.